Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/OnDemand/Utils/Common.hs

```diff
@@ -489,11 +491,13 @@ mkFulfillmentV2 mbDriver mbDriverStats ride booking mbVehicle mbImage mbTags mbP
   where
     driverInfo = forM (liftM2 (,) mbDriver mbDriverStats) $ \(driver, driverStats) -> do
       dPhoneNum <- SP.getPersonNumber driver >>= fromMaybeM (InternalError "Driver mobile number is not present in OnUpdateBuildReq.")
+      dAlternatePhoneNum <- SP.getPersonAlternateNumber driver >>= fromMaybeM (InternalError "Driver alternate mobile number is not present in OnUpdateBuildReq.")
```

Comment:
We should not throw error because of many of drivers will not have alternate number